src/components/resizable_container/resizable_panel.stories.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { css } from '@emotion/react';
+import { faker } from '@faker-js/faker';
+
+import {
+  disableStorybookControls,
+  moveStorybookControlsToCategory,
+} from '../../../.storybook/utils';
+import { EuiResizableContainer } from './resizable_container';
+import { EuiResizablePanel, EuiResizablePanelProps } from './resizable_panel';
+
+faker.seed(42);
+
+const meta: Meta<EuiResizablePanelProps> = {
+  title: 'Layout/EuiResizableContainer/EuiResizablePanel',
+  component: EuiResizablePanel,
+  argTypes: {
+    mode: {
+      control: 'radio',
+      description:
+        'For use with collapsible panels. Will only be applied when used within EuiResizableContainer. View EuiResizableContainer stories for an example.',
+      options: [undefined, 'collapsible', 'main', 'custom'],
+    },
+  },
+  args: {
+    minSize: '0px',
+    scrollable: true,
+    hasShadow: false,
+    borderRadius: 'none',
+    color: 'transparent',
+    paddingSize: 'm',
+    wrapperPadding: 'none',
+    // for quicker/easier QA
+    grow: false,
+    hasBorder: false,
+  },
+};
+disableStorybookControls(meta, ['panelRef']);
+moveStorybookControlsToCategory(
+  meta,
+  [
+    'color',
+    'borderRadius',
+    'grow',
+    'hasBorder',
+    'hasShadow',
+    'paddingSize',
+    'panelRef',
+  ],
+  'EuiPanel props'
+);
+
+export default meta;
+type Story = StoryObj<EuiResizablePanelProps>;
+
+export const Playground: Story = {
+  args: {
+    children: faker.lorem.sentences(5),
+    initialSize: 100,
```

Comment:
Not a blocking change just a suggestion! It was a little hard for me to see the resize indicator all the way over on the right on my screen, this might help
```suggestion
    initialSize: 50,
```